src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -187,6 +207,34 @@ void hwm_EvaluationCompleted(object sender, EvaluationCompletedEventArgs e)
                     SetCurrentWarning(NotificationLevel.Moderate, Properties.Resources.RunCompletedWithWarningsMessage);
                 }
             }
+            else
+            {
+                if (Model.ScaleFactorChanged)
+                {
+                    SetCurrentWarning(NotificationLevel.Error, Properties.Resources.RunCompletedWithScaleChangeAndErrorsMessage);
+                }
+                else
+                {
+                    SetCurrentWarning(NotificationLevel.Error, Properties.Resources.RunCompletedWithErrorsMessage);
+                }
+            }
+
+            SetFooterItems(hasWarnings, hasErrors);
+        }
+
+        /// <summary>
+        /// Updates the Info, Warning and Error footer notification items
+        /// </summary>
+        /// <param name="hasWarnings"></param>
+        /// <param name="hasErrors"></param>
+        private void SetFooterItems(bool hasWarnings, bool hasErrors)
+        {
+            List<FooterNotificationItem> footerItems = new List<FooterNotificationItem>(); //TODO : change to 3 when Info is implemented
+
+            footerItems.Add( new FooterNotificationItem() { NotificationCount = Model.Nodes.Count(n => n.State == ElementState.Error), NotificationImage = "/DynamoCoreWpf;component/UI/Images/error.png" });
+            footerItems.Add( new FooterNotificationItem() { NotificationCount = Model.Nodes.Count(n => n.State == ElementState.Warning || n.State == ElementState.PersistentWarning), NotificationImage = "/DynamoCoreWpf;component/UI/Images/warning_16px.png" });
+
+            FooterNotificationItems = new ObservableCollection<FooterNotificationItem>( footerItems );
```

Comment:
I don't think this is a good pattern, all the old changed events will be no longer be valid, I think it's better to reuse the same collection.

Reply:
Really appreciate the comment, @mjkkirschner! I made a tweak to the code and hopefully addressed your concern. Let me know if that's still iffy and I will follow up to the best of my abilities!